PontoFacil/PontoFacil/Converters/DateTimeToStringConverter.cs

```diff
@@ -0,0 +1,30 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+using Windows.UI.Xaml.Data;
+
+namespace PontoFacil.Converters
+{
+    class DateTimeToStringConverter : IValueConverter
+    {
+        public object Convert(object value, Type targetType, object parameter, string language)
+        {
+            string time = "00:00:00";
+            try
+            {
+                return value != null ? ((DateTime)value).ToString("HH:mm:ss") : time;
+            }
+            catch (Exception)
+            {
+                return time;
```

Comment:
That is some weird with this behaivor, you are using `tyr catch` as a `if-else`, will be better if you havew only one return line after the `try catch` and on the catch fire some mesage about the error.